Example/ios/ScreensExample.xcodeproj/project.pbxproj

```diff
@@ -171,7 +171,6 @@
 				A73D6623F5213E78782B8867 /* Pods-ScreensExample.release.xcconfig */,
 			);
 			name = Pods;
-			path = Pods;
```

Comment:
Please remove this change and it is ready to be merged then 🎉 .

Reply:
@WoLewicki  thanks it's ready to be merged!.